spec/system/admin_user_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Admin crud', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee1) { create(:employee) }
+  let!(:employee2) { create(:employee) }
+  let!(:admin) { create(:admin_user) }
+
+  it 'test admin crud actions' do
+    visit admin_root_path
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: admin.password
+    click_button 'Log in'
+    expect(page).to have_content 'Admin dashboard'
+
+    click_link 'New Kudo'
+    fill_in 'Title', with: 'Title test1'
+    fill_in 'Content', with: 'Content Test1'
+    select employee1.email, from: 'kudo[employee_id]'
+    select employee1.email, from: 'kudo[receiver_id]'
+    click_button 'Create Kudo'
+    expect(page).to have_content 'Kudo was successfully created.'
```

Comment:
Same things, as in Company Value - would be good to make sure, that kudo has been creted, edited and destroyed by checking page contents, not only notifications

Reply:
Fixed.